### PR TITLE
support allInOne otlp grpc and http

### PR DESCRIPTION
**Pull request by dorsegal**
#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [ ] [DCO](https://github.com/jaegertracing/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Commits are [GPG signed](https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification)
- [ ] Chart Version bumped
- [ ] Title of the PR starts with chart name (`[jaeger]` or `[jaeger-operator]`)
- [ ] README.md has been updated to match version/contain new values

### CODE REVIEW

File: charts/jaeger/values.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ fullnameOverride: ""
 allInOne:
   enabled: false
   image: jaegertracing/all-in-one
-  tag: 1.29.0
+  tag: 1.37.0
```
Comment (sparty02) [inline charts/jaeger/values.yaml:19]: Can we just remove this instead?  That will be consistent with the rest of the chart, which defaults image tags to the chart's appVersion

**Review by sparty02 — CHANGES_REQUESTED** [review]

File: charts/jaeger/values.yaml (hunk 1 above)
Comment (jplayout) [reply charts/jaeger/values.yaml:19]: Why not using the tag "latest" ?

File: charts/jaeger/templates/allinone-collector-svc.yaml [hunk 2]
```diff
@@ -21,6 +21,22 @@ spec:
     - name: http-c-binary-trft
       port: 14268
       targetPort: 0
+    {{- if or .Values.allInOne.service.otlp.grpc .Values.allInOne.service.otlp.http }}
```
Comment (naseemkullah) [inline charts/jaeger/templates/allinone-collector-svc.yaml:24]: so if only one of grpc or http are defined, both are set?

**Comment by naseemkullah** [comment]
Thanks @dorsegal , https://github.com/jaegertracing/helm-charts/pull/399/files was merged a while back. Does that fulfill your use case?

**Comment by dorsegal** [comment]
> 

It looks like it almost the same PR. LGTM.

**Pull request closed without merging**